src/lib/date-picker/calendar-table.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  Component,
+  ViewEncapsulation,
+  ChangeDetectionStrategy,
+  Input,
+  EventEmitter,
+  Output
+} from '@angular/core';
+
+
+
+/**
+ * An internal class that represents the data corresponding to a single calendar cell.
+ * @docs-private
+ */
+export class MdCalendarCell {
+  constructor(public value: number, public displayValue: string) {}
+}
+
+
+/**
+ * An internal component used to display calendar data in a table.
+ * @docs-private
+ */
+@Component({
+  moduleId: module.id,
+  selector: 'md-calendar-table, mat-calendar-table',
+  templateUrl: 'calendar-table.html',
+  styleUrls: ['calendar-table.css'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MdCalendarTable {
+  /** The label for the table. */
```

Comment:
Is this the label like`Feb 2017`?  Might want to include an example in the comment to make it clearer. Almost sounds like a placeholder.